brigade.js

```diff
@@ -37,8 +37,8 @@ events.on("push", (e, p) => {
 events.on("check_suite:requested", runSuite);
 events.on("check_suite:rerequested", runSuite);
 events.on("check_run:rerequested", checkRequested);
-events.on("issue_comment:created", handleIssueComment);
-events.on("issue_comment:edited", handleIssueComment);
+events.on("issue_comment:created", (e, p) => Check.handleIssueComment(e, p, runSuite));
```

Comment:
Importing the `Check` object is also needed:

```
const { Check } = require("@brigadecore/brigade-utils");
```

Reply:
Whoops!  Added.  Thanks!